host_configuration_scripts/worker_reboot/linux.sh

```diff
@@ -0,0 +1,11 @@
+echo Shutting down
```

Comment:
nit: This will print even if it is not shutting down in the "already rebooted" case.